### PR TITLE
Fix #28: Comment on pull requests

### LINKED ISSUE #28: Add compatibility comments to pull requests
**Current specification**

When someone submits a pull request to a repository that has YouShouldUse installed, it should go through and add compatibility comments to the lines of code modified in the pull request.

An open pull request with two inline compatibility comments

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: module.js [hunk 1]
```diff
@@ -10,53 +10,70 @@ var parseSource = require('./sourceParse');
...
+            request({
+                url: commitsURL,
+                headers: {
+                    'User-Agent': productName
+                }
+            }, function(error, response, body) {
+                var commits = JSON.parse(body);
+
+                commits.forEach(function(commit, index) {
+                    var thisCommitURL = commit.url;
```
Comment (darkwing) [inline module.js]: Do we need this var?  It's only used once.

File: module.js [hunk 2]
```diff
@@ -10,53 +10,70 @@ var parseSource = require('./sourceParse');
...
+                    request({
+                        url: thisCommitURL,
+                        headers: {
+                            'User-Agent': productName
+                        }
+                    }, function(error, response, body) {
+                        var thisCommit = JSON.parse(body);
```
Comment (darkwing) [inline module.js]: Do we need "this" in the var name?

File: module.js (hunk 1 above)
Comment (openjck) [inline module.js]: Yeah, we can ditch it. `commit.url` is about as self-documenting.

File: module.js [hunk 3]
```diff
@@ -10,53 +10,70 @@ var parseSource = require('./sourceParse');
...
+        var configMetadata = JSON.parse(body);
+        var config = ['last 2 versions'];
 
-            if (configMetadata.content) {
-                var configMetadataContent = new Buffer(configMetadata.content, 'base64').toString();
-                config = configMetadataContent.replace(/\r?\n|\r/g, '').split(/,\s*/);
-            }
+        if (configMetadata.content) {
+            var configMetadataContent = new Buffer(configMetadata.content, 'base64').toString();
+            config = configMetadataContent.replace(/\r?\n|\r/g, '').split(/,\s*/);
```
Comment (darkwing) [inline module.js]: I'd probably add some sort of comment about why you're doing this whitespace stuff.

File: module.js [hunk 4]
```diff
@@ -10,53 +10,70 @@ var parseSource = require('./sourceParse');
...
-            var config = ['last 2 versions'];
+        var configMetadata = JSON.parse(body);
+        var config = ['last 2 versions'];
 
-            if (configMetadata.content) {
-                var configMetadataContent = new Buffer(configMetadata.content, 'base64').toString();
-                config = configMetadataContent.replace(/\r?\n|\r/g, '').split(/,\s*/);
-            }
+        if (configMetadata.content) {
+            var configMetadataContent = new Buffer(configMetadata.content, 'base64').toString();
```
Comment (darkwing) [inline module.js]: var should be hoisted up

Comment (openjck) [inline module.js]: Nice catch! Will do.

File: module.js (hunk 3 above)
Comment (openjck) [inline module.js]: I forgot myself, so yes, definitely need to do that.

File: module.js (hunk 2 above)
Comment (openjck) [inline module.js]: The name `commit` is an outer function parameter. Come to think of it, that's why I used `thisCommitURL`.

Comment (openjck) [inline module.js]: I can look at cleaning up these names generally.

**Comment by openjck** [comment]
Updated!

**Comment by darkwing** [comment]
R+, works great! :)

**Pull request merged**